src/components/pages/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,23 @@
+import PageTitle from "../../common/PageTitle/PageTitle";
+import GallerySet from "../../features/GallerySet/GallerySet";
+import ScrollTopButton from "../../features/ScrollTopButton/ScrollTopButton";
+import galleryData from "./galleryData";
+import useScroll from "../../../hooks/useScroll";
+import useTitle from "../../../hooks/useTitle";
+
+const Gallery: React.FC = () => {
+  useScroll();
+  useTitle("Gallery");
+
+  return (
+    <section>
+      <PageTitle title="Gallery" />
+      {galleryData.map(({ setTitle, works }) => (
+        <GallerySet key={setTitle} title={setTitle} works={works} />
```

Comment:
Nie wiem, jak dla Ciebie, ale dla mnie nazwa `setTitle` jest bardzo myląca. W Reactowym ekosystemie, gdzie `setNazwa` kojarzy Ci się ze zmianą stanu lokalnego, to może być problematyczne.